app/query/processors.go

```diff
@@ -389,6 +405,57 @@ func resolve(ctx context.Context, c *Caller, q *Query, url string) (*ljsonrpc.Cl
 	return &claim, err
 }
 
+// preflightHookClaimSearch patches tag parameters of RPC request to support scheduled and unlisted content.
+func preflightHookClaimSearch(_ *Caller, ctx context.Context) (*jsonrpc.RPCResponse, error) {
+	query := GetFromContext(ctx)
+	params := query.ParamsAsMap()
+	if hasSource, ok := params["has_source"].(bool); ok && hasSource {
+		notTags, _ := params["not_tags"].([]string)
+		anyTags, _ := params["any_tags"].([]string)
+		if !hasAnyTag(params["any_tags"], ClaimTagPrivate, ClaimTagUnlisted) {
```

Comment:
hasAnyTags takes 2 params, 3 are being passed here. So the hasAnyTag check doesn't seem to work.

Reply:
you can also just pass anyTags instead of later converting tags to string?

Reply:
so maybe the main issue is that it's not extracting anyTags and notTags correctly to begin with?